classes/helper/HelperForm.php

```diff
@@ -92,6 +94,23 @@ public function generate()
                         unset($this->fields_form[$fieldset_key]['form']['input'][$key]);
                     }
                     switch ($params['type']) {
+                        case 'switch':
+                            $switch_values = $params['values'];
+                            if (isset($switch_values) && !empty($switch_values)) {
```

Comment:
```suggestion
                            if (!empty($params['values'])) {
```
So if `$switch_values` is empty, nothing occurs, it's not going to the default part? :thinking: 

Then assign the `$params['values']` to `$switchValues`, otherwise it will failing on modern PHP instances. (Undefined index 'values' blablabla).

Reply:
> So if $switch_values is empty, nothing occurs, it's not going to the default part? thinking


This means that the developer did not understand the documentation 😅
What should be the behavior in this case ? Maybe we should ask the PM team ? 🤔
Nice @PierreRambaud  !!

Reply:
ping @PrestaShop/prestashop-product-team :)

Reply:
Hi there! Ideally, `$switch_values` should not be empty but in case no values are defined, I think it makes more sense to have _Yes_ and _No_ as default values. :-)

Reply:
Done !